src/main/kotlin/calculator/domain/Translator.kt

```diff
@@ -0,0 +1,46 @@
+package calculator.domain
+
+import calculator.vo.Token
+import calculator.vo.Tokens
+
+const val DEFAULT_DELIMITER_1 = ','
+const val DEFAULT_DELIMITER_2 = ':'
+
+object Translator {
+    fun run(input: String?): Tokens {
+        if (input.isNullOrBlank()) {
+            return Tokens(listOf(Token()))
+        }
+        val result = translateToList(input)
+        val tokens = translateToToken(result)
+        return Tokens(tokens)
+    }
+
+    private fun translateToList(input: String): List<String> {
+        var result = translateWithCustom(input)
+
+        if (result.isEmpty()) {
+            result = translateWithDefault(input)
+        }
+        return result
+    }
+
+    private fun translateToToken(result: List<String>) = result.asSequence().map { it.toToken() }.toList()
```

Comment:
데이터가 많은 경우가 아니라면 asSequence 없이 아래처럼 사용하는것도 고려해보세요.

```suggestion
    private fun translateToToken(result: List<String>) = result.map { it.toToken() }.toList()
```